packages/react/src/components/Storage/FileUploader/FileUploader.tsx

```diff
@@ -34,6 +33,13 @@ export function FileUploader({
     Tracker = FileUploader.Tracker,
   } = components;
 
+  if (!acceptedFileTypes || !level) {
+    // eslint-disable-next-line no-console
+    console.warn(
+      'You must include the level and acceptedFileNames props to use the file uploader!'
```

Comment:
New warning to users

Reply:
I believe you have to initialize the logger first:

```
const logger = new Logger('AmplifyUI:Auth');

logger.warning('You must include the level and acceptedFileNames props to use the file uploader');
```

Reply:
I'd also:
1. put the logger above this react component so that it doesn't reinitialize on every render, and
2. consider removing exclamation mark from the logger message. Maybe we can also change it to "`FileUploader` requires level and acceptedFileNames props" so that it's immediately clear the error is about FileUploader.

Reply:
Updated [9e9b76c](https://github.com/aws-amplify/amplify-ui/pull/2998/commits/9e9b76c84a1d805e7b995a19fc0b311da69db08a)